doc/findinfiles.rst

```diff
@@ -1,27 +1,58 @@
-:orphan:
+####
+Find
+####
 
-#############
-Find in Files
-#############
-
-The **Find in Files** pane allows to you perform full-text, recursive content search in a user-defined scope, with advanced features to filter your results.
+The **Find** pane allows you to search the content of files in a user-defined location, with advanced features to filter your results.
```

Comment:
```suggestion
The **Find** pane allows you to search the content of plain text files in a user-defined location, with advanced features to filter your results.
```

Just to make clear that this doesn't look inside Word files or something similar.

Reply:
```suggestion
The **Find** pane allows you to search the content of text files in a user-defined location, with advanced features to filter your results.
```

I would simplify this to just "text files", since to many users plain text could imply free-form text (`.txt`) rather than source code, CSV, and other more structured formats.

Reply:
Agreed with @ccordoba12 on making it clearer but with @CAM-Gerlach on making it simpler, I will apply the second one.

Reply:
Agreed.